packages/react-dom/src/events/ReactDOMEventReplaying.js

```diff
@@ -412,16 +413,23 @@ function attemptExplicitHydrationTarget(
 
 export function queueExplicitHydrationTarget(target: Node): void {
   if (enableSelectiveHydration) {
-    const updateLanePriority = getCurrentUpdatePriority();
+    // TODO: This will read the priority if it's dispatched by the React
+    // event system but not native events. Should read window.event.type, like
+    // we do for updates (getCurrentEventPriority).
+    const updatePriority = getCurrentUpdatePriority();
     const queuedTarget: QueuedHydrationTarget = {
       blockedOn: null,
       target: target,
-      lanePriority: updateLanePriority,
+      priority: updatePriority,
```

Comment:
It comes full circle - this was the original name of the field when it was the scheduler priority, which we had to keep alongside lanePriority.